app/src/main/java/io/github/jbellis/brokk/git/GitRepo.java

```diff
@@ -2163,9 +2163,11 @@ public void close() {
      */
     public static void initRepo(Path root) throws GitAPIException, IOException {
         logger.info("Initializing new Git repository at {}", root);
-        Git.init().setDirectory(root.toFile()).call();
-        logger.info("Git repository initialized at {}.", root);
-        ensureBrokkIgnored(root);
+        try (var git = Git.init().setDirectory(root.toFile()).call()) {
+            logger.info("Git repository initialized at {}.", root);
+            ensureBrokkIgnored(root);
+            git.commit().setAllowEmpty(true).setMessage("Initial commit").call();
```

Comment:
We require at least [one branch](https://github.com/BrokkAi/brokk/blob/432ff88a7d72f78e55e055ce62da3ac42106a08d/app/src/main/java/io/github/jbellis/brokk/git/GitRepo.java#L76) to exist in the repo.